### ISSUE TITLE
HTML form element is missing 'rel' attribute in dom types.

### ISSUE THREAD

**Issue by evert** [microsoft/TypeScript#40940]
**TypeScript Version:**  master

**Search Terms:** html form rel

**Code**

```ts
// In TSX, with React:

`<form rel="foo" />`
```

**Expected behavior:** No Error

**Actual behavior:** Yes Error

PR is on the way

**Comment by evert** [comment]
PR got closed, looks like those files are generated. I don't know exactly what my next step would be to fix this.

Should I close this and re-open here: https://github.com/microsoft/TypeScript-DOM-lib-generator ?

**Comment by andrewbranch** [comment]
1. Those typings don’t apply to React
2. While `rel` seems to be a valid attribute on the `form` tag in HTML, it is apparently [not represented on HTMLFormElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement). Seems weird to me, but 🤷 

Proof:

```js
const div = document.createElement('div')
div.innerHTML = '<form rel="whatever"></form>'
div.children[0].tagName // "FORM"
div.children[0].rel // undefined
```

<img width="336" alt="image" src="https://user-images.githubusercontent.com/3277153/95261103-a8bc9f00-07de-11eb-87a6-9969a935648d.png">

**Comment by evert** [comment]
What rabbit hole would you suggest I dive into to get this fixed?

**Comment by andrewbranch** [comment]
I might be underthinking it, but it seems like you just want to add the property [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2de6b3bd78b6cdcaa7eaf582213ac63351c3ad20/types/react/index.d.ts#L2019), for React typings.
